app/models/katello/concerns/redhat_extensions.rb

```diff
@@ -41,11 +41,7 @@ def construct_name(family)
       end
 
       def kickstart_repos(host)
-        distribution_repos = distribution_repositories(host)
```

Comment:
I removed this changed which I added in another PR since I changed `distribution_repositories` to return an empty relation here

Reply:
:thumbsup: